<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MatrixWidget</class>
 <widget class="QWidget" name="MatrixWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1115</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="enterMatrixLabel">
       <property name="text">
        <string>Введите матрицу</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QTableWidget" name="tableMatrixWidget"/>
     </item>
     <item row="3" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QCheckBox" name="modeCheckBox">
         <property name="text">
          <string>Альтернативный режим ввода</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="resultLabel">
         <property name="text">
          <string>Результат: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="getResultButton">
         <property name="text">
          <string>Результат</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QPlainTextEdit" name="MatrixPlainEdit"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
